functions/functionsv2/imagemagick/imagemagick_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package imagemagick
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+)
+
+func TestBlurOffensiveImages(t *testing.T) {
+	projectID := os.Getenv("GOLANG_SAMPLES_PROJECT_ID")
```

Comment:
nit: There is `internal.SystemTest` that other samples use. It will skip the test if this is not set as well as provide context for your test.

Reply:
I've filed https://github.com/GoogleCloudPlatform/golang-samples/issues/2566, and assigned it to myself to update the tests.